scripts/virus-scan-s3-bucket.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+"""Virus scan the contents of an S3 bucket, optionally from a given date onwards.
+Requires a clamd backend to be provided (e.g. https://hub.docker.com/r/mkodockx/docker-clamav/)
+
+You'll need to configure the required AWS environment variables, normally either AWS_PROFILE, or
+AWS_ACCESS_KEY_ID & AWS_SECRET_ACCESS_KEY.
+
+Example:
+    ./scripts/virus-scan-s3-bucket --since 2018-01-01T00:00:00Z digitalmarketplace-documents-production-production
```

Comment:
It's somewhat bothering me that you have an optional argument before the required bucket argument. Could we swap these round so it looks pretty? (sorry)

Reply:
Interesting. I consider optional parameters after the required ones as ugly, for scripts. 🤔

Reply:
Ah yes, fair point. Let's keep it as is